cluster-provision/k8s/1.17/provision.sh

```diff
@@ -2,6 +2,9 @@
 
 set -ex
 
+# Need to have the latest kernel
+dnf update -y --nobest kernel
```

Comment:
Why `--nobest` is used?
Could you explain this and the reasoning for all this change in the commit message?

Reply:
There seem to be a missing step in `./cluster-provision/centos8`, where one can add generic, OS related changes (and not lane related). Such changes seem to fit there more than here. I think this is what @oshoval mentioned above.

Reply:
Yep, thanks

Reply:
Where do we make any modifications to the centos image itself in cluster-provision/centos8?

Reply:
> Where do we make any modifications to the centos image itself in cluster-provision/centos8?

We do not at the moment, that is the “missing step”.
I guess it is an action for a separate PR, but this one reveals the need.

Reply:
cluster-provision/centos8/Dockerfile
what about adding it in this file or in the scripts it runs ?
(either on other PR)

Reply:
Let's fix the current problem right now and then think about how can we expand this to a proper solution.

Reply:
sgtm, just rebase is needed, there are couple PRs on fire today that changing hashes